docs/source/getting_started/getting_started_hard_way.rst

```diff
@@ -0,0 +1,269 @@
+  cd iroha
+
+.. hint:: If you have installed the prerequisites with Docker, you don't need to clone Iroha again
+
+
+Building Iroha
+^^^^^^^^^^^^^^
+To build Iroha, use those commands
+
+.. code-block:: shell
+
+  cmake -H. -Bbuild;
+  cmake --build build -- -j$(nproc)
+
+.. note::  On macOS ``$(nproc)`` variable does not work. Check number of logical cores with ``sysctl -n hw.ncpu`` and put it explicitly in the command above, e.g. ``cmake --build build -- -j4``
+
+CMake parameters
+^^^^^^^^^^^^^^^^
+
+We use CMake to build platform-dependent build files. It has numerous of the flags for configuring the final build. Note that beside the listed params cmake's vars can be useful as well. Also as long as this page can be deprecated (or just not complete) you can browse custom flags via ``cmake -L``, ``cmake-gui``, or ``ccmake``.
+
+.. hint::  You can specify parameters at the cmake configuring stage (e.g cmake -DTESTING=ON).
+
+Main params
+"""""""""""
+
++--------------+-----------------+---------+------------------------------------------------------------------------+
+| Parameter    | Possible values | Default | Description                                                            |
++==============+=================+=========+========================================================================+
+| TESTING      |      ON/OFF     | ON      | Enables or disables build of the tests                                 |
++--------------+                 +---------+------------------------------------------------------------------------+
+| BENCHMARKING |                 | OFF     | Enables or disables build of the benchmarks                            |
++--------------+                 +---------+------------------------------------------------------------------------+
+| COVERAGE     |                 | OFF     | Enables or disables coverage                                           |
++--------------+                 +---------+------------------------------------------------------------------------+
+| SWIG_PYTHON  |                 | OFF     | Enables or disables libraries and native interface bindings for python |
++--------------+                 +---------+------------------------------------------------------------------------+
+| SWIG_JAVA    |                 | OFF     | Enables or disables libraries and native interface bindings for java   |
++--------------+-----------------+---------+------------------------------------------------------------------------+
+
+Packaging specific params
+"""""""""""""""""""""""""
+
++-----------------------+-----------------+---------+--------------------------------------------+
+| Parameter             | Possible values | Default | Description                                |
++=======================+=================+=========+============================================+
+| ENABLE_LIBS_PACKAGING |      ON/OFF     | ON      | Enables or disables all types of packaging |
```

Comment:
I don't really understand why parameter ENABLE_LIBS_PACKAGING — Enables or disables all types of packaging, maybe there is something missing

Reply:
@l4l